packages/react-devtools/package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "react-devtools",
+  "productName": "React Developer Tools",
   "version": "2.0.12",
   "description": "Use react-devtools outside of the browser",
+  "main": "./packages/react-devtools/app.js",
```

Comment:
Meaning I should point to `app.js` relative to the directory it's in (which I've updated... whoops) or that I should be pointing this to `index.js`?